app/views/posts/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>New Post</h1>
+
+<%= form_with(model: @post, url: user_posts_path(@post.author)) do |form| %>
+  <%= form.label :title %>
+  <%= form.text_field :title %>
+
+  <%= form.label :text %>
+  <%= form.text_area :text %>
+
+  <%= form.submit %>
```

Comment:
- [ ] Kindly fix the error for this form, and add buttons for users to navigate your app properly, currently, I had to type the URL to call the view (form).

__Note:__ I didn't test other forms part from the posts new because I can't create a comment or a like without a post.


![form](https://github.com/ridabensalem/blog-app/assets/53124548/d424781e-bcb2-46dc-9bdb-e85a3e100c63)